### PR TITLE
create-the-first-part-of-account-page

### PR DESCRIPTION
**Pull request by IsraaAmani**

### CODE REVIEW

File: Account/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section class="cover">
+        <div class="background" >
+            <div class="icons">
+            <img class="img1" src="img/chevron-circle-left.png">
```
Comment (yaqootturman) [inline Account/index.html:14]: don't use order for naming classes, such as img1, img2

File: Account/style.css [hunk 2]
```diff
@@ -0,0 +1,215 @@
+@media (max-width: 360px) {
```
Comment (yaqootturman) [inline Account/style.css:1]: Why? 
we build mobile first, so no neet for make a media query for mobile

File: Account/style.css [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+  body {
```
Comment (yaqootturman) [inline Account/style.css:2]: The design is not so good, you should make it more suitable
image

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Comment by nebal96** [comment]
good

**Pull request closed without merging**